differences.md

```
@@ -0,0 +1,114 @@
+## Analysis of crec2.txt and comparison to src/ project for Export, Pagination, Doc ID Control, and Comparison (Revised Again - DOCX Import & Export Clarified)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)